### ISSUE TITLE
Save map project as HydroShare resource does not work

### ISSUE THREAD

**Issue by dtarb** [hydroshare/hydroshare#1259]
From apps.hydroshare.org if I select the "Save as..." button to save map project as HydroShare resource I get the error "An unknown/unexpected error was encountered.  Project not saved."

This is somewhat critical because this is live on the operational site and a sort of obvious thing that a user may want to do.

**Comment by shawncrawley** [comment]
The issue has been fixed. Behind the scenes it was a "Referencing an undefined variable" error. That was a foolish mistake on my part, because I had tested everything as thoroughly as I could (including the Save as button) and it was working. However, last second before pushing the changes I thought "instead of hard-coding that variable in multiple places, I'll give it a variable name and define/assign a value to it in only one place." I changed the hard-coded variables to the new variable name, but did not ever defined/assigned a value to it, pushed, and did not re-test since "it was a small/easy fix." Goes to show that no change is small enough to overshadow the need for re-testing. 

For the fix I added one line to define/assign a value to the variable. It should be working properly now. I tested it myself this time.
